demos/python_demos/action_recognition/action_recognition_demo/result_renderer.py

```diff
@@ -86,7 +86,8 @@ def render_frame(self, frame, logits, timers, frame_ind):
 
         cv2.imshow("Action Recognition", frame)
 
-        if cv2.waitKey(1) & 0xFF == ord('q'):
+        key = cv2.waitKey(1) & 0xFF
+        if key == ord('q') or key == ord('Q') or key == 27:
```

Comment:
You can simplify this to `if key in {...}:`

Reply:
Updated